src/warnet/project.py

```diff
@@ -209,7 +209,11 @@ def check_installation(tool_info: ToolInfo) -> ToolStatus:
             inquirer.List(
                 "platform",
                 message=click.style("Which platform would you like to use?", fg="blue", bold=True),
-                choices=["Minikube", "Docker Desktop"],
+                choices=[
+                    "Minikube",
+                    "Docker Desktop",
+                    "No Backend (Interacting with remote cluster already deployed)",
```

Comment:
Might be worth prompting, or at least hinting to the user they will need to run `warnet auth <kubeconfig for remote cluster>` if choosing the "No Backend" option. Otherwise, they might finish the setup and then nothing works. The auth command is also nice in that it makes sure that kubectl is switched over to use the "warnet" context.

Reply:
Could even run it for them?

Reply:
i like it, added